config/initializers/01_redis.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+module Octobox
+  REDIS_URL = ENV.fetch("REDIS_URL", "redis://localhost:6379")
+end
```

Comment:
Should be part of the [Configurator](https://github.com/octobox/octobox/blob/d76bf25df157ddff096dee938f0728fb595496d2/lib/octobox/configurator.rb) class with all the rest of the configuration; or are you having trouble having that loaded before the Sidekiq initializer?